model/src/gateway/payload/stage_instance_create.rs

```diff
@@ -0,0 +1,20 @@
+use crate::channel::StageInstance;
+use serde::{Deserialize, Serialize};
+use std::ops::{Deref, DerefMut};
+
+#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
```

Comment:
Typically structs shouldn't implement `Hash` since this wouldn't be used as e.g. something like a HashMap key

Reply:
I copied these from another gateway payload, if it's eventually a goal to remove hash from every payload then i think that should be in another pr